src/backend/joanie/core/templatetags/extra_tags.py

```diff
@@ -41,6 +43,19 @@ def list_key(items: list[dict], key: str):
     return [item[key] for item in items]
 
 
+@register.filter
+def iso8601_to_date(value: str, arg: str = None) -> str:
+    """
+    Custom tag filter to format an ISO 8601 datetime string into
+    a formatted date string.
```

Comment:
Maybe add more details in the docstring of how to use the template tag : {{ value|iso8601_to_date" }} in a template django html ? Perhaps, it could be useful in a near future.

WDYT ?